CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### [1.5.6](https://github.com/rokkreslincom/cordova-plugin-advanced-imagepicker/compare/V1.5.5...V1.5.6) (2022-01-25)
```

Comment:
Remove these changes completely. This is done automatically!

Reply:
It was a mistake, sorry.